data/raw/keybindings.json

```
@@ -1852,6 +1858,13 @@
     "name": "Toggle activate/examine",
     "bindings": [ { "input_method": "keyboard_char", "key": "!" }, { "input_method": "keyboard_code", "key": "1", "mod": [ "shift" ] } ]
   },
+  {
+    "type": "keybinding",
+    "id": "APPLY",
+    "category": "MEDICAL",
+    "name": "Apply or Use Item",
```

```suggestion
    "name": "Apply and/or Use Item",
```
making this change to see if I can force tests to run